folio-spring-i18n/src/main/java/org/folio/spring/i18n/config/TranslationConfiguration.java

```diff
@@ -0,0 +1,29 @@
+package org.folio.spring.i18n.config;
+
+import java.util.Locale;
+import lombok.Getter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * An {@link org.springframework.context.annotation.Configuration Configuration} bean to hold the
+ * location of the current translation directory and default fallback locale.
+ */
+@Getter
+@Configuration
+public class TranslationConfiguration {
+
+  private String translationDirectory;
+
+  private Locale fallbackLocale;
+
+  @Autowired
+  public TranslationConfiguration(
+    @Value("/translations/") String translationDirectory,
```

Comment:
Are there any pros to assigning constant string in such way?

Reply:
In case a module needs to override the values, they can create a new bean and override it.  I've also been using it substantially in testing.